libs/theme/src/variables.css

```
@@ -19,4 +19,14 @@
   --background-color: white;
   --background-contrast-color: black;
   --error-color: red;
+
+  /* Font sizes */
+  --small-font-size: 12px;
+  --medium-font-size: 16px;
+  --large-font-size: 22px;
+
+  /* Other custom variable */
+  --border-color: #bebebe;
+  --border-dark-color: #7e7e7e;
+  --bottom-border-thin: 2px;
```

I would use `size` or `width` instead because it is hard to image how `thin` can be measured while is it pretty clear with the alternatives that I gave you
```suggestion
  --bottom-border-size: 2px;
```

---

I actually don't need it, forgot to delete.